packages/kbn-i18n/GUIDELINE.md

```diff
@@ -385,6 +388,12 @@ Splitting sentences into several keys often inadvertently presumes a grammar, a
 
   If this group of sentences is separated it’s possible that the context of the `'it'` in `'close it'` will be lost.
 
+### Large paragraphs
+
+Try to avoid having to use large paragraphs of text. These are difficult to maintain and often need slight changes when the information they portray becomes out of date. 
+
+If you have absolutely no choice, we can split paragraphs into a _few_ i18n chunks whenever it makes sense to do so. Similar principles of splitting sentences apply here. Each chunk on it's own should convey the context of the information.
```

Comment:
```suggestion
If you have absolutely no choice, you can split paragraphs into a _few_ i18n chunks whenever it makes sense to do so. Similar principles of splitting sentences apply here. Each chunk on its own should convey the context of the information.
```

> Each chunk on it's own should convey the context of the information

nit: Had to read this a couple times to understand the intent. Maybe something along these lines?  "Chunks should be split at logical points to ensure they contain enough context to be intelligible on their own."

Reply:
@lukeelmers Thanks for the suggestion!